benchmark-memory.gemspec

```diff
@@ -1,12 +1,12 @@
-require File.expand_path('../lib/benchmark/memory/version', __FILE__)
+require File.expand_path('lib/benchmark/memory/version', __dir__)
 
 Gem::Specification.new do |spec|
   spec.name          = 'benchmark-memory'
   spec.version       = Benchmark::Memory::VERSION
   spec.authors       = ['Michael Herold']
   spec.email         = ['michael.j.herold@gmail.com']
 
-  spec.summary       = 'Benchmark-style memory profiling for Ruby 2.1+'
+  spec.summary       = 'Benchmark-style memory profiling for Ruby 2.4+'
```

Comment:
:thinking: I'm thinking about this. The gem should work on older versions (down to 2.1). Perhaps I shouldn't have added a version here in the first place.

Reply:
I just removed it in this PR.